admin/partials/wp-security-bp-admin-display.php

```diff
@@ -14,12 +14,115 @@
 ?>
 
 <!-- This file should primarily consist of HTML with a little bit of PHP. -->
-<h1>OLAKEASE</h1>
 <?php
 
-$users = get_users( array( 'include' => array( 2, 3 ) ) );
-foreach ( $users as $user ) {
-    if ( true ) {
-        var_dump( $user );
+class WP_Security_BP_Admin_Get_Users {
```

Comment:
I suggest the following changes to this class:
1 - Name it **WP_Security_BP_Users** as it might do more things in the future than just get users.
2 - Put it in a separate file inside the **includes** folder.
3 - Use DocBlock comments as a best practice, you can copy/paste/change from other classes: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/